### ISSUE TITLE
Documentation Missing Function Parameters

### ISSUE THREAD

**Issue by Druidpeter** [nicrohobak/AnsiGL#2]
Hiya, Nicrohobak! I'm currently working on integrating AnsiGL into a personal game engine, and I'm running into issues with your documentation. Basically, I can't use the documentation *.txt files you provide because often the methods you have in those files don't include their parameters, return values, and usage information. As a result, I have to hunt through the source code very frequently to figure out exactly how to use your api.

Some updated documentation that included the above aforementioned information would make my life much, much easier. Sorry for the trouble, and thank you for your time!

**Comment by nicrohobak** [comment]
You're absolutely correct in that the docs are incomplete.  I can do nothing but apologize for this, really.  Sorry about that!

To expand though, this is largely because things like this were still evolving quite a bit at the time those were written and because of that, at this point, they're really more of a quick-reference index than real documentation.  I also have to refer to the source for info, but once I re-familiarize the "docs" become more helpful again.

It has been a while since I've made large changes, so it may be worth the effort at this point to expand on them a bit more...it should probably be "safe enough" at this point that it won't be rendered obsolete too quickly.  I don't know yet when I'll have time, but I'll see if I can make a pass at it...or I may just end up providing a doxyfile for Doxygen or something instead...I'll take a look and see if I can get something a little more helpful at least.

Also, I will gladly accept contribution from anyone willing to assist...so if you do end up creating some notes or anything that is better than what I have here, don't hesitate to share! :)

**Comment by Druidpeter** [comment]
Mind if I take a crack at getting some doxygen documentation up? I've already forked your repo, so I should be able to get a pull request to you sometime decent, assuming I get anywhere with it.

**Comment by nicrohobak** [comment]
By all means...I'd be happy to add it.

**Comment by Druidpeter** [comment]
Hey, Nicrohobak.

So, I made this issue a long time ago and promptly disappeared. Not cool on my part. I won't bore you with the details as to what happened. Needless to say, my semester is over in two weeks. If this project is still up and running, I'd like to come back and try working with this code-base again. 

(Long Story Short: Life blew up for me for a while).
